lib/screens/dashboard.dart

```diff
@@ -16,21 +16,35 @@ class Dashboard extends StatelessWidget {
       appBar: AppBar(
         title: Text(_tituloAppBar),
       ),
-      body: Container(
-        color: Colors.white,
-        child: Column(
-          mainAxisAlignment: MainAxisAlignment.spaceBetween,
-          children: <Widget>[
-            Quadro(
-              altura: 400.0,
-              caminhoImagem: _imagemDashboard,
+      /**TODO para manter o dashboard com o mesmo visual e
+       * com comportamentos de scroll tive que usar esse exemplo
+       * que peguei na página da documentação,
+       * é necessário realizar esse procedimento completo?
+       * */
+      body: LayoutBuilder(
+        builder: (context, viewportConstraints) {
+          return SingleChildScrollView(
+            child: ConstrainedBox(
+              constraints: BoxConstraints(
+                  minHeight: viewportConstraints.maxHeight,
```

Comment:
Why are you constraining to the viewport?
You need this widget to occupy the whole screen, is that it?
If so, using just the Column below isn't enough?

Reply:
Yes, the Column fill whole screen, but I see a warning/bug saying that Widgets is not reached when I restart the App with keyboard opened, then I added the SingleChildScrollView for avoid the bug. Using only SingleChildScrollView, the space between items doenst work well, because the Column doesnt filled whole screen, then I added this constraints to keep the same visual.